db/migrate/20190909232448_create_game_table.rb

```diff
@@ -0,0 +1,8 @@
+class CreateGameTable < ActiveRecord::Migration[5.2]
+  def change
+    create_table :crazy_eight_games do |t|
+      t.text :deck_id
```

Comment:
```suggestion
      t.text :deck_api_id
```
Being a little more verbose here would help avoid ambiguity, since when I first saw this line I was confused since by ActiveRecord convention 'something_id' is an Integer for a Foreign Key. This change would have to be applied to all files.